docs/md_merge.py

```diff
@@ -0,0 +1,357 @@
+import argparse
+import os
```

Comment:
Nit: Following subsequent suggestions to always use the `pathlib.Path` functionality instead of mixing, we can remove that import

Reply:
Updated to get rid of `os` altogether.